lua/package-info/modules/core/show.lua

```diff
@@ -48,9 +47,8 @@ local get_package_metadata = function(current_package_version, outdated_dependen
 end
 
 local set_virtual_text = function(dependencies, outdated_dependencies)
-    local is_file_package_json = check_if_package_json()
-
-    if not is_file_package_json then
+    -- may be an unnecessary check
```

Comment:
This is needed due to how the flow goes.

1. `npm outdated` command is ran
2. Virtual text is set

If you leave the buffer before the `npm outdated` command finishes, it's going to put virtual text in any buffer you are in, so this prevents that.

Reply:
Ohhh makes sense now! Thanks!